bucket/macrocreator.json

```
@@ -0,0 +1,29 @@
+{
+    "version": "5.3.2",
+    "description": "Free Automation Tool and Script Generator based on AutoHotkey language.",
+    "homepage": "https://www.macrocreator.com",
+    "license": "GPL-3.0-or-later",
+    "url": "https://github.com/Pulover/PuloversMacroCreator/releases/download/v5.3.2/PuloversMacroCreator-Portable.zip",
+    "hash": "03691b301a2d55742b5bbf1c6145a2963c0287428d58535c9b1b3596397db4d9",
+    "architecture": {
+        "64bit": {
+            "extract_dir": "MacroCreatorPortable\\x64\\MacroCreator"
+        },
+        "32bit": {
+            "extract_dir": "MacroCreatorPortable\\x86\\MacroCreator"
+        }
+    },
+    "shortcuts": [
```

https://github.com/Pulover/PuloversMacroCreator/blob/master/MacroCreator.ahk#L152-L154

```suggestion
    "pre_install": "Copy-Item \"$persist_dir\\*\" \"$dir\" -Include 'UserEmailAccounts.ini', 'UserGlobalVars.ini' -ErrorAction SilentlyContinue",
    "uninstaller": {
        "script": "Copy-Item \"$dir\\*\" \"$persist_dir\" -Include 'UserEmailAccounts.ini', 'UserGlobalVars.ini' -ErrorAction SilentlyContinue -Force"
    },
    "shortcuts": [
```